### PR TITLE
added smooth scrolling and fixed footer position and added contact info into the footer

### PR DESCRIPTION
**Pull request by akashxlr8**
I have fixed the footer position (earlier it was not sticking to the bottom)  and added contact info into the footer (using on-page navigation).

Before
image

After
image

Also added smooth scrolling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,158 +1,168 @@
...
-description: "World's first Mentorship platform at your fingertips.Mentro, brings professional guidance from industry experts working in companies like Google, Microsoft, Amazon, Jio, Flipkart, GoJek, CRED and more at a personalised level for you."
+description: "World's first Mentorship platform at your fingertips.Mentro, brings professional guidance from industry
+experts working in companies like Google, Microsoft, Amazon, Jio, Flipkart, GoJek, CRED and more at a personalised level
+for you."
 image: "/assets/images/mentro_logo.png"
-keywords: 
-    - "Mentro"
-    - "mentorship"
+keywords:
+- "Mentro"
```
Comment (LoopThrough-i-j) [inline index.html:8]: please fix the formatting

Comment (akashxlr8) [reply index.html]: Fixed it. Please review.

**Comment by LoopThrough-i-j** [comment]
Can you push 2 commits? It is appreciated to have all white space and indentation changes in 1 commit and the real code change in one commit.

**Comment by akashxlr8** [comment]
Hello, I have reverted all the white space and indentation changes. Can you please review it again.

**Pull request merged**